app/views/components/_datetime_fields.html.erb

```diff
@@ -1,6 +1,7 @@
 <%
  prefix = prefix
  field_name = field_name
+ id = id
```

Comment:
These 3 fields locally make no sense

Reply:
Broke pipeline on testing, dropped commit. Issue can be created for investigation